packages/paste-core/layout/stack/src/index.tsx

```diff
@@ -91,12 +91,13 @@ const Stack: React.FC<StackProps> = ({children, orientation, spacing, ...props})
   );
   const stackStyles = React.useMemo(() => getStackStyles(orientation), [orientation]);
   const childMargins = React.useMemo(() => getStackChildMargins(orientation, spacing), [orientation, spacing]);
+  const keySeed = useUIDSeed();
 
   return (
     <Box {...safelySpreadBoxProps(props)} {...stackStyles}>
       {validChildren.map((child, index) => {
         return (
-          <Box {...(childrenCount !== index + 1 ? childMargins : null)} key={useUID()}>
+          <Box {...(childrenCount !== index + 1 ? childMargins : null)} key={keySeed(`${index}-stack`)}>
```

Comment:
Is this change working as intended? From what I've gathered, this will return `val-${index}-stack` and not provide "uid" functionality :( https://github.com/thearnica/react-uid/blob/master/src/uid.ts#L23

This may work better as `keySeed(child)`. The API is confusing and I plan to improve this in our UID lib hopefully.

Reply:
It seems to work, but I prefer `keySeed(child)` 👍

Reply:
OK, back to using an index in the string of a seed. It will create a unique seed for each stack, and within each stack the children are unique based on index.

If I used `keySeed(child)` it causes an infinite loop due to too many state updates when the children are dynamic.